nix/crates.nix

```diff
@@ -122,7 +122,7 @@ in {
 
     cargo-test-int = craneLib.cargoNextest (commonWorkspace
       // {
-        buildInputs = commonWorkspace.buildInputs ++ [octez pkgs.cacert];
+        buildInputs = commonWorkspace.buildInputs ++ [pkgs.iana-etc octez pkgs.cacert];
```

Comment:
What's this for?

Reply:
nix thing for the test to work, @johnyob added

Reply:
Allows the query for `/etc/services` to work within the nix sandbox. This is required by cohttp lib, that octez client depends on.